apps/ui/src/components/trace/traces.tsx

```diff
@@ -35,7 +34,7 @@ export function SchemaTraces() {
   const navigate = useNavigate();
   const params = useParams();
   const [searchParams] = useSearchParams();
-  const [searchModal, setSearchModal] = useState(false);
+  const [search, setSearch] = useState("");
```

Comment:
I'm not completely sure how the default state here works. Does it match `""` against all trace names and hence return all traces?

Reply:
Yeah if it's an empty string it will return all traces.